Frontend.py

```diff
@@ -65,38 +83,51 @@ def ListNotes():
         print(note)
     
     print(f"{end_color}")
+    main()
     
 def ViewNotes(): 
     print(f"{bold_text}{purple_text}\nView Notes{end_color}")
     print(f"{blue_text}")
-    read_file = input("Input the Markdown File you wish to view: ")
+    read_file = input(f"{underline_text}Input the Markdown File you wish to view: {end_color}{blue_text}")
     if ".md" not in read_file:
         read_file = f"{read_file}.md"
     
     read_file = f"Notes/{read_file}"
         
     if not os.path.isfile(read_file):
-        print(f"{read_file} Does Not Exist")
+        print(f"{read_file} Does Not Exist{end_color}{blue_text}")
         print("Use the AddNotes() function if you wish to create this file")
         exit()
     else:    
         with open(read_file, 'r') as r:
             note_content = r.read()
             print(note_content)
-    
+        
     print(f"{end_color}")
-    
+    main()
+
+def Introduction():
+    print(f"{bold_text}{purple_text}__________________________________________________________________")
+    print("__________________________________________________________________\n")
+    print("Welcome to Noteworthy - The #1 Notetaking Application For Markdown")
+    print("__________________________________________________________________")
+    print("__________________________________________________________________")
+    
+    {end_color}
+    main()
+
 def main(): 
-    print(f"{bold_text}{purple_text}Welcome to Noteworthy - The #1 Notetaking Application For Markdown{end_color}\n")
-    print(f"{blue_text}")
+    print(f"{blue_text}{underline_text}")
```

Comment:
Right now the way our app has been designed seems a bit weird to me. Of course I'll merge cause these changes are fire but just hear me out.

In my opinion it would be awesome if in the future we could make the frontend a Python executable file called `noteworthy` that could be called with command line arguments similar to how this app is being run. The way we have it setup with `main` feels awkward to me because it's really a software that runs continuously until you close it; which it doesn't really need to do. I guess I just like the idea of our tool being a command line tool like git or vim rather than an application.

@nherman71 @tferrelCOW @jteferi thoughts?